### PR TITLE
Feature/cld 345

### PR DESCRIPTION
**Pull request by twosdai**
Includes the following updates from acceptance criteria:

 - Update references to "MarkLogic" to state "MarkLogic Server"
 - Define initialized and uninitialized in the beginning of the Readme
 -- explain the user would be prompted for credentials
 - Include a Prerequisites section defining the expected starting conditions before running
 MarkLogic in Docker
 -- Docker CLI is mentioned and external documentation is linked.
 -- For instance, need to run on a full browser - not on phone or tablet
 - replace PLACEHOLDER_IMAGE text with the current image ID
 - before <insert admin username> <insert admin password> in code explain these need to be replaced as well.
 - All code examples with debugging info, or expected state, must filter correctly and contain expected results.
 - Include instructions on how to verify MarkLogic has come online correctly
 -- Check logs (give command examples with some)
 -- Go to browser check port 8001
 - Include Debugging instructions
 - Link to Docker documentation on persistent data directory prior to section.
 - Creation and Management of the .yaml files in the clustering section is explained.
 - "docker swarm join --token random-string-of-characters-generated-by-docker-swarm-command should have an additional quote
 - Link to Overlay network explanation for docker in multi-vm section.
 - Generalize vm section to go from vm: 1 - n
 - Include a cleanup section and commands.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,35 +102,38 @@ $ docker run -d -it -p 8000:8000 -p 8001:8001 -p 8002:8002 \
...
 
-MarkLogic Docker containers are configured via a set of environment variables.
+MarkLogic Server Docker containers are configured via a set of environment variables.
 
 
 | env var                       | value                           | required                          | default   | description                                        |
 | ------------------------------- | --------------------------------- | ----------------------------------- | ----------- | ---------------------------------------------------- |
-| MARKLOGIC_INIT                | true                            | no                                | <br/>     | when set to true, will initialize server           |
+| MARKLOGIC_INIT                | true                            | no                                | <br/>     | when set to true, will initialize MarkLogic           |
 | MARKLOGIC_ADMIN_USERNAME      | jane_doe                        | required if MARKLOGIC_INIT is set | n/a       | set MarkLogic admin user                           |
```
Comment (vitalykorolev) [inline README.md:117]: I think MarkLogic in description column should be MarkLogic Server. e.g MarkLogic Server admin password.

File: README.md [hunk 2]
```diff
@@ -72,35 +102,38 @@ $ docker run -d -it -p 8000:8000 -p 8001:8001 -p 8002:8002 \
...
 
 ## Clustering
 
-MarkLogic Docker containers ship with a small set of scripts, making it easy to create clusters. Below are three examples for creating MarkLogic clusters with Docker containers. The first two use [Docker compose](https://docs.docker.com/compose/) scripts to create one-node and three-node clusters. The third example demonstrates a container setup on separate VMs.
+MarkLogic Server Docker containers ship with a small set of scripts, making it easy to create clusters. Below are three examples for creating MarkLogic Server clusters with Docker containers. The first two use [Docker compose](https://docs.docker.com/compose/) scripts to create one-node and three-node clusters. The third example demonstrates a container setup on separate VMs.
 
 The credentials for admin user are configured via Docker secrets, and are stored in mldb_admin_username.txt and mldb_admin_password.txt files. Make sure the yaml files for Docker compose have the desired image label and volume path (~/data/MarkLogic). All of these scripts were tested with version 20.10 of Docker.
 
 ### Single node MarkLogic on a single VM
```
Comment (vitalykorolev) [inline README.md:134]: Should be "MarkLogic Server"

File: README.md (hunk 1 above)
Comment (twosdai) [reply README.md]: 0d4b25a Updated

**Review by vitalykorolev — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -24,28 +30,50 @@ MarkLogic documentation is available at [http://docs.marklogic.com](https://docs
...
 ## Using this Image
 
-To create an uninitialized MarkLogic server with [Docker CLI](https://docs.docker.com/engine/reference/commandline/cli/), run this command:
+Optionally we can either create an initialized or an uninitialized MarkLogic Server. 
 
-```
-$ docker run -d -it -p 8000:8000 -p 8001:8001 -p 8002:8002 \
-     PLACEHOLDER-FOR-DOCKER-IMAGE:DOCKER-TAG
-```
+For an initialized MarkLogic Server, admin credentials are required to be passed while creating the docker container. The docker container will have MarkLogic Server installed and initialized. MarkLogic Server will have databases and app servers created. A security database will be created to store user data,roles and other security information. MarkLogic Server credentials, passed as env params while running a container, will be stored as admin user in the security database. These admin credentials can be used to access MarkLogic Server Admin interface on port 8001 and other app servers with respective ports.
```
[the remark this answers is not in the document]
Comment (twosdai) [reply README.md]: Really good catch, didn't even notice the non-escaped commands. took me a second. 

Anyway its updated: fd76bdc

File: README.md [hunk 4]
```diff
@@ -24,28 +30,50 @@ MarkLogic documentation is available at [http://docs.marklogic.com](https://docs
...
+Example run 
+```
+> docker run -d -it -p 8000:8000 -p 8001:8001 -p 8002:8002 \                                                                                                                                                  
+     -e MARKLOGIC_INIT=true \
+     -e MARKLOGIC_ADMIN_USERNAME=admin \
+     -e MARKLOGIC_ADMIN_PASSWORD=Areally!PowerfulPassword1337 \
+     store/marklogicdb/marklogic-server:10.0-8.1-centos-1.0.0-ea2
+8834a1193994cc75405de27d6985eba632ee1e9a1f4519dac6ff833cecb9abb6
+```
+Wait about a minute for MarkLogic Server to initialize before checking the ports. To verify the successful installation and initialization, login to MarkLogic Server admin interface using admin credentials provided while running the container, this is achieved by navigating to http://localhost:8000. After the initial login, you should see app servers and databases created on the host and admin user in the security database. Optionally you can check logs on admin interface from the logs tab.
```
[the remark this answers is not in the document]
Comment (twosdai) [reply README.md:57]: Yeah there may be some documentation with pictures I can link to with basic UI images.

[the remark this answers is not in the document]
Comment (twosdai) [reply README.md:57]: If I can't find any I think we can nix it.

File: README.md [hunk 5]
```diff
@@ -1,8 +1,14 @@
+## Prerequisites
```
[the remark this answers is not in the document]
Comment (twosdai) [reply README.md:1]: Yeah good call.

[the remark this answers is not in the document]
Comment (twosdai) [reply README.md:1]: 2b5959f  Updated

**Comment by twosdai** [comment]
@vitalykorolev I capitalized docker throughout the Readme: a9a3376

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (vitalykorolev) [reply README.md:59]: I like it. It'll keep readme shorter for those who are familiar and provide additional details for those who may be new to MarkLogic.

[the remark this answers is not in the document]
Comment (twosdai) [reply README.md]: 1c1589e  Let me know what you think of this.

**Pull request merged**